l1-contracts/contracts/bridge/L1NativeTokenVault.sol

```diff
@@ -154,91 +174,29 @@ contract L1NativeTokenVault is IL1NativeTokenVault, IL1AssetHandler, Ownable2Ste
         chainBalance[_chainId][l1Token] += amount;
 
         // solhint-disable-next-line func-named-parameters
-        _bridgeMintData = abi.encode(amount, _prevMsgSender, _l2Receiver, getERC20Getters(l1Token), l1Token);
-        // solhint-disable-next-line func-named-parameters
+        _bridgeMintData = DataEncoding.encodeBridgeMintData(
+            amount,
+            _prevMsgSender,
+            _l2Receiver,
+            getERC20Getters(l1Token),
+            l1Token
+        ); // solhint-disable-next-line func-named-parameters
```

Comment:
do we need to disable it here? maybe we could make the event accept named params